backend/src/services/user.ts

```diff
@@ -44,6 +47,20 @@ async function getMenteeId(pairingId: string): Promise<string> {
   return pairing.menteeId;
 }
 
+async function updateMentorFCMToken(fcmToken: string, userId: string) {
+  console.log("FCM Token: ", fcmToken);
```

Comment:
Let's consolidate the update FCM token functions into 1 function instead of 2.  When calling the function user should specific role as well. Use that to determine whether to search the Mentee or Mentor collection.